invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/state/reducer.js

```diff
@@ -26,6 +33,25 @@ export default (state = initialState, action) => {
         error: action.payload,
         hasError: true,
       };
+    case DELETE_SUCCESS:
+      return {
+        ...state,
+        isLoading: false,
+        data: {
+          hits: state.data.hits.filter(
+            hit => hit.location_pid !== action.locationId
```

Comment:
:+1:

Reply:
Is this because ES is still not up-to-date? Can we add a comment or something? It might look strange at first look.